src/components/SecondaryNav.tsx

```diff
@@ -40,25 +40,23 @@ const SecondaryNav = ({setLimit, activeLink, setActiveLink, user}: SecondaryNavP
   });
 
   return (
-    <div>
-      <div className="bg-darkestGrey py-14 md:py-16">
+    <div>{console.log(activeLink)}
```

Comment:
Is this a feature we really need ? I see that in the past we do not have this feature.